static/multifile-service.ts

```diff
@@ -133,7 +133,7 @@ export class MultifileService {
 
         const zip = await JSZip.loadAsync(f);
 
-        await Promise.all(zip.map(async (relativePath, zipEntry) => {
+        await Promise.all(zip.forEach(async (relativePath, zipEntry) => {
```

Comment:
This wouldn't work, nothing is returned with forEach, so there would be nothing to await. Luckily no one actually relies on it currently, so maybe we can change this by removing `await Promise.all()`.

Reply:
Indeed, removed the await.